### PR TITLE
Fixed 0.4 seconds not showing up in exposure collection

### PR DESCRIPTION
**Pull request by mondoman712**

### CODE REVIEW

File: src/libs/collect.c [hunk 1]
```diff
@@ -1249,7 +1249,7 @@ static void list_view(dt_lib_collect_rule_t *dr)
         break;
 
       case DT_COLLECTION_PROP_EXPOSURE: // exposure
-        g_snprintf(query, sizeof(query), "SELECT CASE WHEN (exposure <= 0.5) "
+        g_snprintf(query, sizeof(query), "SELECT CASE WHEN (exposure < 0.4) "
```
[the remark this answers is not in the document]
Comment (TurboGit) [reply src/libs/collect.c]: Can you test with latest fix I just pushed? If not working, can you provide the image showing the issue? Thanks.

[the remark this answers is not in the document]
Comment (mondoman712) [reply src/libs/collect.c]: The problem is that it switches from fraction to decimals at 0.5, but 0.4 won't show up as a fraction. You need to switch at 0.4 instead. I have attached an image with shutter speed and exposure time set to 0.4" although you could use
`exiftool -ExposureTime="0.4" -ShutterSpeedValue="0.4" x.jpg` to create one.
test

[the remark this answers is not in the document]
Comment (mondoman712) [reply src/libs/collect.c]: Yes and there's no 0.4" in the list.

**Comment by TurboGit** [comment]
Should be fixed in another commit.

**Pull request closed without merging**